android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUIText.java

```diff
@@ -472,6 +493,9 @@ public void handleKeyboard(KrollDict d)
 		int autocorrect = InputType.TYPE_TEXT_FLAG_AUTO_CORRECT;
 		int autoCapValue = 0;
 
+		// Initialize keyboard
+		keyboard = d;
+		
 		if (d.containsKey(TiC.PROPERTY_AUTOCORRECT) && !TiConvert.toBoolean(d, TiC.PROPERTY_AUTOCORRECT, true)) {
 			autocorrect = 0;
 		}
```

Comment:
##### LINE 548
```Java
...
case TEXT_AUTOCAPITALIZATION_ALL:
    autoCapValue = InputType.TYPE_TEXT_FLAG_CAP_CHARACTERS;
    autoCapAll = true;
    break;
```

set `autoCapAll = false;` for other AUTOCAPITALIZATION conditions